docs/user/quickstart.rst

```diff
@@ -211,6 +211,7 @@ Note: Custom headers are given less precedence than more specific sources of inf
 
 Furthermore, Requests does not change its behavior at all based on which custom headers are specified. The headers are simply passed on into the final request.
 
+Note: All header values must be a ``string``, bytestring (on Python 2), or unicode (on Python 3).
```

Comment:
This isn't quite accurate: they're allowed to be bytestrings or unicode on any version of Python, it's just that passing unicode is rarely a good idea. =)